website/src/content/blog/2025-03-31-typespec-1-0-release/type_spec_1_0_release.md

```diff
@@ -0,0 +1,160 @@
+---
+slug: 2025-03-31-typespec-1-0-release
+title: "Introducing TypeSpec 1.0: APIs at the Speed of Thought"
+image: ./TypeSpec_1_0_release.png
+description: "TypeSpec 1.0 is here! Accelerate your API development by automatically generating server-side code, client SDKs, schemas, and documentation directly from concise API models, enabling you to focus on innovation rather than implementation details."
```

Comment:
Is the target to release that on the 31st? If so this feels deceiving if we say 1.0 is here but we only release the rc

Reply:
What if I called it TypeSpec 1.0-RC and made clear it was a release candidate?

Reply:
yeah Im fine with that, also happy to republish again/update the article when we we move out of the rc